simp_le.py

```diff
@@ -1435,6 +1435,26 @@ def finalize_order(client, order):
     return finalized_order
 
 
+def poll_and_answer(client, authorizations, roots):
+    """Poll authorization status and answer challenge if required"""
+    # pylint: disable=no-member
```

Comment:
I tried removing this on my local machine and pylint didn't complain. What's the error you're seeing?

Reply:
```
simp_le.py:1474:15: E1101: Instance of 'Field' has no 'status' member (no-member)
simp_le.py:1476:15: E1101: Instance of 'Field' has no 'status' member (no-member)
```

https://travis-ci.com/github/buchdag/simp_le/jobs/305433258

It might have been fixed on a more up to date version of `pylint`.

Reply:
Could you try removing it and see if it still flags? I'd prefer not to have this in here if we can avoid it, since it could mask real bugs down the line.

Reply:
Seems to be ok now : https://travis-ci.com/github/buchdag/simp_le/jobs/331238335

Reply:
Great, I'll go ahead and merge.